.travis.yml

```diff
@@ -158,16 +162,44 @@ matrix:
           packages:
             - gcc-9
             - g++-9
+      script: .travis/script.sh
     - env: BUILD_TARGET="posix-ncp-spi" VERBOSE=1
       os: linux
       compiler: gcc
+      script: .travis/script.sh
     - env: BUILD_TARGET="toranj-test-framework" VERBOSE=1
       os: linux
       compiler: gcc
+      script: .travis/script.sh
     - env: BUILD_TARGET="toranj-test-framework" VERBOSE=1 TORANJ_POSIX_APP_RCP_MODEL=1
       os: linux
       compiler: gcc
       python: "2.7"
+      script: .travis/script.sh
     - env: BUILD_TARGET="osx" VERBOSE=1
       os: osx
       language: generic
+      script: .travis/script.sh
+    - stage: lint
+      env: BUILD_TARGET="py-pretty-check"
+      os: linux
+      python: "2.7"
+      script: .travis/script.sh
+    - env: BUILD_TARGET="pretty-check"
+      os: linux
+      addons:
+        apt:
+          sources:
+            - llvm-toolchain-xenial-6.0
+            - ubuntu-toolchain-r-test
+          packages:
+            - clang-format-6.0
+      script: .travis/script.sh
+    - env: BUILD_TARGET="size-report"
+      os: linux
+      compiler: gcc
+      script: .travis/script.sh VERBOSE=1
+      if: type = pull_request
+stages:
```

Comment:
nit: prefer to put `stages` above `jobs` since the former groups the latter.

Reply:
moved above jobs.